### PR TITLE
chore: cache node_modules

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,10 @@ before_install:
 install:
   - npm install
 
+cache:
+  directories:
+    - "node_modules"
```
Comment (selfup) [inline .travis.yml]: Will the cache invalidate when package-lock changes? Never knew you could do this in travis :tada:

Just curious 🙏

Comment (DanielRuf) [reply .travis.yml]: > Will the cache invalidate when package-lock changes?

Yes, see https://docs.travis-ci.com/user/caching/

Comment (selfup) [reply .travis.yml]: Oh that's quite neat!

**Comment by jorgebucaran** [comment]
Looks good!

**Comment by jorgebucaran** [comment]
Thank you, @DanielRuf! 🎉

**Pull request merged**